crates/sui-indexer/src/apis/read_api.rs

```diff
@@ -338,6 +339,20 @@ where
         events_guard.stop_and_record();
         events_resp
     }
+
+    fn get_dynamic_fields_loaded_objects(
```

Comment:
Dynamic fields are a a bit of an abstraction over dynamically loaded child objects, I'd try to keep the naming a bit separate if possible

Reply:
I guess my point is that everywhere else refers to this map as `loaded_child_objects`, then suddenly dynamic fields pops up here